cutil/url.py

```diff
@@ -74,4 +78,7 @@ def is_video(url):
 
 def is_media(url):
```

Comment:
@asmitks Do you think it will be better to have a separate argument `strict: bool` to switch between checking only mimetype and otherwise ?

Reply:
yeah sure will add it